Implemented static UI for PO review page(#85zrgy1rw)

disha1202 opened the pull request:
### Related Issues
<!--  Put related issue number which this PR is closing. For example #123 -->

Closes #117 

### Short Description and Why It's Useful
<!-- Describe in a few words what is this Pull Request changing and why it's useful -->


### Screenshots of Visual Changes before/after (If There Are Any)
<!-- If you made any changes in the UI layer, please provide before/after screenshots -->
![Screenshot from 2022-12-28 12-32-59](https://user-images.githubusercontent.com/58461801/209772154-abf90f83-4f6a-4a05-88c5-98a71e4067ee.png)

![Screenshot from 2022-12-28 16-40-05](https://user-images.githubusercontent.com/58461801/209807552-a37875eb-c209-43e9-b626-a68139073331.png)

![Screenshot from 2022-12-28 16-40-16](https://user-images.githubusercontent.com/58461801/209807568-6dfa5b0b-66a6-48d2-9762-16253d9ca76d.png)

![Screenshot from 2022-12-28 16-40-30](https://user-images.githubusercontent.com/58461801/209807587-6311f804-7f16-41ec-ac88-ad9e35a652d8.png)



**IMPORTANT NOTICE** - Remember to add changelog entry


### Contribution and Currently Important Rules Acceptance
<!-- Please get familiar with following info -->

- [ ] I read and followed [contribution rules](https://github.com/hotwax/import#contribution-guideline)

shashwatbangar commented:
On PODetail page, PO2 is sticking to left border.
Rest UI looks good to me.

disha1202 commented:
> On PODetail page, PO2 is sticking to left border. Rest UI looks good to me.
Fixed and added some more UI elements.

ymaheshwari1 commented on src/components/BulkAdjustmentModal.vue:
```diff
@@ -0,0 +1,108 @@
+<template>
+    <ion-header>
```
Improve indentation throughout.

dt2patel commented on src/components/BulkAdjustmentModal.vue:
```diff
@@ -0,0 +1,108 @@
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content class="ion-padding">
```
Remove ion padding

dt2patel commented on src/components/DateTimeParseErrorModal.vue:
```diff
@@ -0,0 +1,104 @@
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content class="ion-padding">
```
Remove padding here and everywhere else.

disha1202 replied:
Removed.

dt2patel reviewed (changes requested):
Need to fix indentation and review element selector practices. I think you can do one round of improvements yourself @disha1202

disha1202 commented:
> Need to fix indentation and review element selector practices. I think you can do one round of improvements yourself @disha1202

Fixed indentation and updated markup.
Please take a look.
@dt2patel
